tests/dll/nimhcr_integration.nim

```diff
@@ -1,5 +1,5 @@
 discard """
-  disabled: "openbsd"
+  disabled: "true"
```

Comment:
I hope that's temporary?

Reply:
I don't know, I don't understand why my minor refactoring breaks this test and I invested 2 hours already. Be my guest.

Reply:
possibly related to:
* https://github.com/nim-lang/Nim/issues/11974
* https://github.com/nim-lang/Nim/issues/15081
* https://github.com/nim-lang/Nim/pull/16223 

I also spent time to fix an HCR issue (which enabled that integration test in the first place) eg in https://github.com/nim-lang/Nim/pull/16329.

It could be just 1 underlying bug (HCR related). However it'd be really really nice to fix it instead of disabling it

Reply:
I can reproduce it too on my local machine but that doesn't help.